Composer/packages/client/src/pages/botProject/table-view.tsx

```diff
@@ -0,0 +1,736 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
```

Comment:
This is a huge component, I was going through to comment but there are some repetitive comments, please break down this big component to the smaller component and compose them together in the TableView.